src/components/standalone/security/ips/IpsDisabledRules.vue

```diff
@@ -0,0 +1,221 @@
+  })
+})
+
+const sortKey = ref<keyof Rule>('id')
+const sortDescending = ref(false)
+const { sortedItems } = useSort(filteredRules, sortKey, sortDescending, {})
+
+const pageSize = ref(10)
+const { currentPage, paginatedItems } = useItemPagination(sortedItems, {
+  itemsPerPage: pageSize
+})
+
+// FIXME: when types from library are fixed, use proper type
+const onSort = (payload: any) => {
+  sortKey.value = payload.key
+  sortDescending.value = payload.descending
+}
+
+const disablingRule = ref(false)
+function handleSave() {
+  listDisabledRules()
+  changes.getChanges()
+  disablingRule.value = false
+}
+
+function dropDownActions(rule: Rule): NeDropdownItem[] {
+  return [
+    {
+      id: 'delete',
+      label: t('common.delete'),
+      icon: 'trash',
+      iconStyle: 'fas',
+      danger: true,
+      action: () => {
+        ruleToEnable.value = rule
+      }
+    }
+  ]
+}
+const ruleToEnable = ref<Rule>()
+function handleEnabled() {
+  listDisabledRules()
+  changes.getChanges()
+  ruleToEnable.value = undefined
+}
+</script>
+
+<template>
+  <div class="space-y-8">
+    <div class="flex flex-wrap items-start justify-between gap-4">
+      <p class="max-w-lg">{{ t('standalone.ips.disabled_rules_description') }}</p>
+      <IpsEnabledBadge />
+    </div>
+
+    <NeInlineNotification
+      v-if="error"
+      :description="t(getAxiosErrorMessage(error))"
+      :title="t('standalone.ips.error_loading_disabled_rules')"
+      kind="error"
+    />
+    <NeSkeleton v-if="loading" :lines="10" />
+    <template v-else-if="rules.length > 0">
+      <div class="flex flex-col flex-wrap justify-between gap-4 md:flex-row">
+        <NeTextInput v-model="filter" :placeholder="t('common.filter')">
+          <template #prefix>
+            <FontAwesomeIcon :icon="faMagnifyingGlass" aria-hidden="true" class="h-4 w-4" />
+          </template>
+        </NeTextInput>
+        <NeButton v-if="rules.length > 0" kind="secondary" size="lg" @click="disablingRule = true">
+          <template #prefix>
+            <FontAwesomeIcon :icon="faXmarkCircle" aria-hidden="true" class="h-4 w-4" />
+          </template>
+          {{ t('standalone.ips.disable_rule') }}
+        </NeButton>
+      </div>
+      <NeTable
+        :ariaLabel="t('standalone.ips.rules_table')"
+        :skeleton-columns="7"
+        :skeleton-rows="5"
+        :sortDescending="sortDescending"
+        :sortKey="sortKey"
+        card-breakpoint="xl"
+      >
+        <NeTableHead>
+          <NeTableHeadCell column-key="description" sortable @sort="onSort">
+            {{ t('standalone.ips.rule_description') }}
+          </NeTableHeadCell>
+          <NeTableHeadCell column-key="id" sortable @sort="onSort">
+            {{ t('standalone.ips.rule_id') }}
+          </NeTableHeadCell>
+          <NeTableHeadCell>
+            <!-- no header for actions -->
+          </NeTableHeadCell>
+        </NeTableHead>
+        <NeTableBody>
```

Comment:
Looks like the empty state for "No rules found with current filter" is missing. A good example of handling the two empty states inside the table is here: https://github.com/NethServer/nethsecurity-ui/blob/main/src/components/standalone/firewall/nat/NatHelpersTable.vue#L64-L90